tests/BUILD

```diff
@@ -4,9 +4,23 @@ sse_copts = ['-msse4']
 omp_copts = ['-fopenmp']
 omp_linkopts = ['-lgomp']
 
+windows_copts = [
+    "/arch:AVX",
+    "/std:c++14",
```

Comment:
What is the reason to use C++14 instead of C++11?

Reply:
C++14 is also used in the TFQ builds, which are what we need to align with for these tests. I'll double-check these with their team to be certain this was intended, but for now we'll stick with this.

(As an aside, and possibly a note to my future self: `/std:c++11` is not a recognized flag when building with Kokoro, as it uses C++11 by default.)